server/src/server.ts

```diff
@@ -531,3 +569,13 @@ const getMarkdownContent = (documentation: string): LSP.MarkupContent => ({
   // Passed as markdown for syntax highlighting
   kind: 'markdown' as const,
 })
+
+function getCommandOptions(name: string, word: string): string[] {
+  // TODO: The options could be cached.
+  const options = Process.spawnSync(Path.join(__dirname, '../src/get-options.sh'), [
+    name,
+    word,
+  ])
+
+  return options.stdout.toString().split('\t')
```

Comment:
It seems we should trim and filter the empty results here:
```suggestion
  return options.stdout
    .toString()
    .split('\t')
    .map(l => l.trim())
    .filter(l => l.length > 0)
```